recipe/meta.yaml

```
@@ -0,0 +1,78 @@
+{% set version = "1.1.46" %}
+{% set name = "whylogs" %}
+{% set sha256 = "f73307893960ed8460544d3b3efeb53b53a943d7c0caf3a749279690f034f51d" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py<37]
+  script: {{ PYTHON }}  -m pip install --no-deps --ignore-installed --no-build-isolation .
+
+requirements:
+  host:
+    - python
+    - pip
+    - wheel
+    - setuptools
+    - poetry
```

```suggestion
    - poetry-core
```
When possible, prefer poetry-core over poetry as it brings in less dependencies.